docker-compose-develop.yml

```diff
@@ -3,7 +3,7 @@ services:
   develop:
     build: .
     ports:
-      - "9000:9000"
+      - "30505:30505"
```

Comment:
This port is not exposed by the dockerfile, so this wont work. You have a similar issue on a different PR for a different MS.

Reply:
It works. Docker compose will expose that port.